network-api/networkapi/buyersguide/templates/primary_nav.html

```diff
@@ -0,0 +1,55 @@
+<div id="primary-nav-container">
+  <div class="wrapper-burger">
+    <div class="menu-container">
+      <div class="narrow-screen-menu">
+        <div class="narrow-screen-menu-background">
+          <div class="narrow-screen-menu-container">
+            <div class="social d-flex flex-column">
+              <a class="social-button body-large social-button-fb" href="#TODO">Facebook</a>
+              <a class="social-button body-large social-button-twitter" href="#TODO">Twitter</a>
+              <a class="social-button body-large social-button-email" href="#TODO">Email</a>
+              <a class="social-button body-large social-button-link" href="#TODO">Copy link</a>
+            </div>
+            <div class="donate">
+              <div class="container text-center">
+                <div class="row">
+                  <div class="col">
+                    <h3 class="h3-heading">Help us keep the Internet healthy</h3>
+                    <p>Mozilla is a nonprofit organization fighting for a healthy interent.</p>
+                    <a class="btn btn-donate m-0 w-100" href="https://donate.mozilla.org">Donate</a>
```

Comment:
does this need the same utm query args as the footer?

Reply:
@Pomax I'm not sure - this chunk of code was already in the codebase. I only moved it to this file. Let's file a followup ticket to get clarification then.

Reply:
filed https://github.com/mozilla/foundation.mozilla.org/issues/1959